scripts/devops_tasks/common_tasks.py

```diff
@@ -177,21 +202,35 @@ def process_glob_string(glob_string, target_root_dir, additional_contains_filter
     # however, if there are multiple packages being built, we should honor the omission list and NOT build the omitted
     # packages
     else:
-        allowed_package_set = remove_omitted_packages(collected_directories)
+        allowed_package_set = remove_omitted_packages(collected_directories, ommit_type)
         logging.info("Target packages after filtering by omission list: {}".format(allowed_package_set))
 
         pkg_set_ci_filtered = filter_for_compatibility(allowed_package_set)
         logging.info("Package(s) omitted by CI filter: {}".format(list(set(allowed_package_set) - set(pkg_set_ci_filtered))))
 
         return sorted(pkg_set_ci_filtered)
 
-def remove_omitted_packages(collected_directories):
-    return [
+def remove_omitted_packages(collected_directories, ommit_type = OmmitType.Default):
+
+    packages = [
         package_dir
         for package_dir in collected_directories
         if os.path.basename(package_dir) not in OMITTED_CI_PACKAGES
     ]
 
+    ommit_regression = lambda x: "nspkg" not in x and "mgmt" not in x and os.path.basename(x) not in META_PACKAGES and os.path.basename(x) not in TRACK1_CLIENT_PACKAGES
```

Comment:
Normally I would be unhappy with using basename specifically (instead of parsing setup) but given that we're _returning package directories anyway_ I'm good with this :)